qkernel/src/interrupt/x86_64/mod.rs

```diff
@@ -28,6 +28,14 @@ use super::super::threadmgr::task_sched::*;
 use super::super::MainRun;
 use super::super::SignalDef::*;
 use super::super::SHARESPACE;
+cfg_cc! {
+    use super::super::qlib::cc::sev_snp::{VCResult,SVMExitDef,snp_active,C_BIT_MASK};
```

Comment:
using relative paths was a mistake and will be fixed later. Perhaps consider ditch this pattern in new code already.